stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.5.1
+version: 5.5.2
```

Comment:
Please bump the chart version to latest. Thank you.

Reply:
@zanhsieh, I've now bumped the version to 5.5.3

Reply:
@zanhsieh, did you literally mean "latest" ? I can do that if it is what you meant.

Reply:
Nay. We should stick on specific version number, not latest. I don't think git or helm will take 'latest'.